File: bubblesort.py

def print_arr(arr):
    for i in arr:
        print(i, end=", ")
    print()


def bubble_sort(arr):
        for i in range(len(arr)-1):
            for j in range(0, len(arr)-1-i):
                if arr[j] > arr[j+1]:
                    arr[j], arr[j+1] = arr[j+1], arr[j]


def main():
    arr = [2,5,1,3,6,8,9,9,4]
    print_arr(arr)
    bubble_sort(arr)
    print_arr(arr)


if __name__ == '__main__':
    main()

File: Bubblesort.cs
using System;

namespace Bubble
{
	class Schort
	{
		static void print_arr(int[] arr)
		{
			for (int i = 0; i < arr.Length; i++)
			{
				Console.Write(arr[i] + ", ");
			}
			Console.Write("\n");
		}

		static void do_bubble(int[] arr)
		{
			Random rndm = new Random();

            
            for (int i = 0; i < arr.Length-1; i++)
			{
                for (int j = 0; j < arr.Length-1-i; j++)
				{
                    if (arr[j] > arr [j+1])
                    {
                        int bittehelfensiemir = arr[j+1];
                        arr[j+1] = arr[j];
                        arr[j] = bittehelfensiemir;
                    }
                }
            }
        }

		static void Main(String[] args)
		{
			int[] arr = {2,5,1,3,6,8,9,9,4};
			print_arr(arr);
			do_bubble(arr);
			print_arr(arr);
		}
	}
}